buildkite/pipeline.yml

```diff
@@ -15,3 +15,10 @@ steps:
 
   - label: ":shipit: Push images"
     command: docker/push
+    
+  - wait
+  
+  - label: ":hammer: Browser tests"
```

Comment:
I think these tests should come before the push

Reply:
I thought we talked about putting them after so they can pull and run the docker image. You'd rather this step built a local image to run?